packages/expo-media-library/ios/MediaLibraryUtilities.swift

```diff
@@ -2,38 +2,6 @@ import ExpoModulesCore
 import Photos
 import CoreServices
 
-func stringify(mediaType: PHAssetMediaType) -> String {
```

Comment:
moved this to `MediaType.init(fromPHAssetMediaType mediaType: PHAssetMediaType)` to prevent the two from going out of sync